packages/hadron-express/src/__tests__/hadronToExpress.ts

```diff
@@ -131,15 +126,19 @@ describe('router config', () => {
       return routesToExpress(testRoute, Container).then(() =>
         request(app)
           .get(`/index`)
-          .then(() => {
-            expect(eventManager.emitEvent.calledOnce).to.equal(true);
-          }),
+          .then(() =>
```

Comment:
especially here I think `async / await ` should improve much the readability

Reply:
As above, I would rather not change all then/catch, but stick to one part. But if You feel like it's a game changer, feel free to do such refactor.